webpack.config.npm.js

```diff
@@ -124,14 +124,6 @@ const parserConfig = Object.assign({}, defaultConfig, {
     calciteAutocompleteParser: [`${JS_ROOT}/parse/sql/calcite/calciteAutocompleteParser.js`],
     calciteSyntaxParser: [`${JS_ROOT}/parse/sql/calcite/calciteSyntaxParser.js`],
 
-    druidAutocompleteParser: [`${JS_ROOT}/parse/sql/druid/druidAutocompleteParser.js`],
```

Comment:
I noticed that we have druid still available for selection [here](https://docs.gethue.com/developer/components/parsers/) and perhaps there is further cleanup needed.  Should we perhaps do that in a separate jira?

Reply:
Will create a docs PR.